std/encoding.d

```diff
@@ -652,7 +662,8 @@ template EncoderInstance(E)
 
 private template GenericEncoder()
 {
-    bool canEncode(dchar c)
+    @safe:
+    bool canEncode(dchar c) @safe
```

Comment:
Is the `@safe` here necessary? Since this is a template member, shouldn't the compiler already infer attributes for it?